.github/workflows/publish-latest.yaml

```diff
@@ -61,13 +62,10 @@ jobs:
         with:
           context: .
           # supported platforms of https://hub.docker.com/_/golang/tags?page=1&name=1.23-alpine3.20
-          # platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x
+          platforms: linux/amd64,linux/arm64
           push: true
           tags: |
-            ghcr.io/open-component-model/ocm:latest
-            ghcr.io/open-component-model/ocm/ocm.software/ocmcli/ocmcli-image:latest
-            ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+            ghcr.io/hilmarf/ocm/ocm.software/ocmcli/ocmcli-image:latest
```

Comment:
what about the "personalised" path `hilmarf`? Is this intentional?

Reply:
argh... nope - did forget about this open PR, when trying out some tests yesterday... unfortunately working on my main branch :-( sorry will fix it